alerts/cluster-etcd-operator/OWNERS

```diff
@@ -0,0 +1,9 @@
+reviewers:
+  - hexfusion
+  - ironcladlou
+  - lilic
+  - marun
+approvers:
+  - deads2k
```

Comment:
Unless @hexfusion disagrees I believe we should add Maru and myself to approvers as well. If its possible of course? :)

Reply:
I can do it now, but once we add the syncing, it's going to be whoever is in the operator repo's owners file.

Reply:
yes

Reply:
I'll add both as approvers. Again just as note: this will be overwritten, as soon as we put the syncing in place.